src/modules/get_institution/app/get_institution_presenter.ts

```diff
@@ -0,0 +1,18 @@
+import { GetInstitutionController } from "./get_institution_controler";
+import { GetInstitutionUsecase } from "./get_institution_usecase";
+
+import { Repository } from "../../../core/repositories/Repository";
+import { HttpRequest } from "../../../core/helpers/http/http_codes";
+
+const InstituteRepository = new Repository({institution_repo: true});
+const UserRepository = new Repository({user_repo: true});
```

Comment:
You instance the const Repository and pass:
```
const Repository = new Repositoy({ user_repo: true, institution_repo: true })
``` 

is no need to create two Repos!